diff --git a/docker/server/Dockerfile b/docker/server/Dockerfile
index 15a6d6e..3f176b6 100644
--- a/docker/server/Dockerfile
+++ b/docker/server/Dockerfile
@@ -5,7 +5,7 @@
 # ===========================================================================================================
 # 0. Builder stage
 # ===========================================================================================================
-FROM openjdk:11-jdk AS builder
+FROM eclipse-temurin:11-jdk-focal AS builder
 
 LABEL maintainer="Netflix OSS <conductor@netflix.com>"
 
@@ -19,19 +19,22 @@ RUN ./gradlew build -x test --stacktrace
 # ===========================================================================================================
 # 1. Bin stage
 # ===========================================================================================================
-FROM openjdk:11-jre
+FROM eclipse-temurin:11-jre-focal
 
 LABEL maintainer="Netflix OSS <conductor@netflix.com>"
 
+ARG start_script
 # Make app folders
 RUN mkdir -p /app/config /app/logs /app/libs
 
 # Copy the compiled output to new image
-COPY --from=builder /conductor/docker/server/bin /app
-COPY --from=builder /conductor/docker/server/config /app/config
+COPY --from=builder /conductor/docker/bin /app
+COPY --from=builder /conductor/docker/config /app/config
 COPY --from=builder /conductor/server/build/libs/conductor-server-*-boot.jar /app/libs
 
 # Copy the files for the server into the app folders
+RUN chmod +x /app/$start_script
+RUN mv /app/$start_script /app/startup.sh
 RUN chmod +x /app/startup.sh
 
 HEALTHCHECK --interval=60s --timeout=30s --retries=10 CMD curl -I -XGET http://localhost:8080/health || exit 1
