diff --git a/docker/server/Dockerfile b/docker/server/Dockerfile
index 15a6d6e..f2584ac 100644
--- a/docker/server/Dockerfile
+++ b/docker/server/Dockerfile
@@ -27,12 +27,8 @@ LABEL maintainer="Netflix OSS <conductor@netflix.com>"
 RUN mkdir -p /app/config /app/logs /app/libs
 
 # Copy the compiled output to new image
-COPY --from=builder /conductor/docker/server/bin /app
-COPY --from=builder /conductor/docker/server/config /app/config
 COPY --from=builder /conductor/server/build/libs/conductor-server-*-boot.jar /app/libs
-
-# Copy the files for the server into the app folders
-RUN chmod +x /app/startup.sh
+# We inject /app/startup.sh using docker mount or k8s configmap file so that it would create the proper /app/config on boot if need
 
 HEALTHCHECK --interval=60s --timeout=30s --retries=10 CMD curl -I -XGET http://localhost:8080/health || exit 1
 
diff --git a/docker/server/Dockerfile b/docker/server/Dockerfile
index f2584ac..6554e6b 100644
--- a/docker/server/Dockerfile
+++ b/docker/server/Dockerfile
@@ -32,5 +32,5 @@ COPY --from=builder /conductor/server/build/libs/conductor-server-*-boot.jar /ap
 
 HEALTHCHECK --interval=60s --timeout=30s --retries=10 CMD curl -I -XGET http://localhost:8080/health || exit 1
 
-CMD [ "/app/startup.sh" ]
-ENTRYPOINT [ "/bin/sh"]
+#CMD [ "/app/startup.sh" ]
+#ENTRYPOINT [ "/bin/sh"]
